src/app/dex/page.tsx

```diff
@@ -17,17 +17,22 @@ import Contract, { errorHandler } from "@/services/api";
 import GGXWallet from "@/services/ggx";
 import TokenDecimals from "@/tokenDecimalsConverter";
 import type { Amount, DetailedOrder } from "@/types";
-import { BN_ZERO } from "@polkadot/util";
+import { BN, BN_ZERO } from "@polkadot/util";
 import { useRouter } from "next/navigation";
 import { Rule } from "postcss";
-import { Suspense, useEffect, useRef, useState } from "react";
+import { Suspense, useEffect, useMemo, useRef, useState } from "react";
 import Loading from "./loading";
+import { toast } from "react-toastify";
 
 type TokenData = TokenWithPrice & {
 	amount: Amount;
 };
 
 export default function Dex() {
+	let mesg = ''
+	const milisecPerYear = useMemo(() => {
+		return new BN(31536000).mul(new BN(1000));
```

Comment:
this looks like something that can be a constant, why is it in `useMemo`?

Reply:
Yes, a constant is a better choice.